### ISSUE TITLE
Cerebro Position Incorrect After Disconnecting & Reconnecting External Displays From Laptop

### ISSUE THREAD

**Issue by jonnyasmar** [KELiON/cerebro#439]
- [x ] I am on the [latest](https://github.com/KELiON/cerebro/releases/latest) Cerebro.app version
- [x ] I have searched the [issues](https://github.com/KELiON/cerebro/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Ubuntu 16.04 w/ Unity
- **Cerebro.app version**: 0.3.2
- **Relevant information from devtools** Not sure, but I don't think dev tools shows any info about the app container.

## Issue

Running Cerebro on Ubuntu 16.04 (w/ Unity), Laptop + 2x 4K monitor set up: [1920x1080] [3840x2160] [3840x2160].

The middle display (4K) is my primary monitor (where the launcher is designated to sit). Upon first installing Cerebro & switching my primary display setting, it was successfully opening in the center of my display. After disconnecting my external displays and reconnecting them, Cerebro now spawns at the top right of my left-most (Laptop) display. I should mention as well that when I was running on just my single laptop display, the position was also incorrect (bottom right).

**Comment by lawgsy** [comment]
I think the entire opening, closing, showing, hiding and positioning of the window needs rewriting. It's a complete mess at present.

**Comment by jonnyasmar** [comment]
Yes. I haven't dug into that bit of the code myself, yet, but it definitely seems that way.

I did find a workaround for this in the meantime -- I've modified the config file manually, lol. However, I think there definitely needs to be some specific handling for multiple displays.

**Comment by oguhpereira** [comment]
Could test to see if the problem remains in current versions. I believe the topic here has been resolved @jonnyasmar @lawgsy
